live-build/variants/internal-dcenter/migration-scripts

```diff
@@ -0,0 +1 @@
+../../misc/migration-scripts
```

Comment:
This shouldn't apply to the `internal-dcenter` variant.

Reply:
I think we could push that as a rule in the potential future work where we have a finer-granularity of the selection of what we generate (e.g. generate only a specific image for a specific variant). Until then it may be simpler to just leave all variants having a similar structure. Does this sound reasonable?